code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm

```diff
@@ -0,0 +1,90 @@
+/datum/antagonist/tiger_fanatic
+	name = "Tiger Cooperative Fanatic"
+	antagpanel_category = "Other"
+	job_rank = ROLE_TIGER_FANATIC
+	show_name_in_check_antagonists = TRUE
+	preview_outfit = /datum/outfit/tiger_fanatic
+	antag_moodlet = /datum/mood_event/focused
+	suicide_cry = "FOR THE HIVE!!"
+	hardcore_random_bonus = TRUE
+	ui_name = "AntagInfoTigerFanatic"
+	var/blessings = 0
+
+/datum/antagonist/tiger_fanatic/greet()
+	. = ..()
+	to_chat(owner.current, span_changeling("You worship the changeling hive!."))
+	to_chat(owner.current, span_changeling("You have detected the pressence of the changeling hive mind, and have smuggled yourself aboard the station."))
+	to_chat(owner.current, span_changeling("This is your one opertunity to make contact with a changeling, you must be assimilated into the hive in order to accend."))
+	to_chat(owner.current, span_changeling("You have a weak connection to the changeling hivemind, your body has been conditioned to make you a perfect offering to the changelings."))
```

Comment:
This should all be in the antag info panel

Reply:
You do think it should also be removed from the greet()?

Reply:
Yes, antag info panel is the replacement for greet

Reply:
done